dask_jobqueue/tests/test_remote_slurm.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+import pytest
+
+from dask_jobqueue.remote_slurm import RemoteSLURMCluster
+
+TEST_API_URL = "http://a-url.com/slurm"
+
+
+def test_end_to_end(requests_mock):
```

Comment:
It would be good to have an async test here too to ensure the `asyncio` functionality also works (required for the Dask Jupyter Lab Extension to function).

Reply:
Just found a `test_job` function to which I can add RemoteSLURMCluster and RemoteSLURMJob. Then async should be fully tested. :v: